packages/ibm-products/src/components/SidePanel/SidePanel.tsx

```diff
@@ -525,7 +525,9 @@ export let SidePanel = React.forwardRef(
         }
       };
       const bodyElement = document.body;
-      if (includeOverlay && open) {
+      const isStorybook = document.getElementById('storybook-sidepanel');
```

Comment:
I think this logic should be kept outside of the actual component. Is there a way we can include this in the stories file?